lib/features.js

```diff
@@ -110,18 +110,21 @@ module.exports = config => {
      * @param recency
      * @returns {boolean}
      */
-    canBypassSiginConfirmation(verified, recency) {
-      let bypass = false
+    canBypassSiginConfirmation(email, verified, recency) {
+      // If sign-in confirmation is forced for an email, it can't be bypassed.
+      if (signinConfirmation.enabled && signinConfirmation.enabledEmailAddresses.test(email)) {
+        return false
+      }
 
       // IP Profiling sets bypass to true if this user has verified a session
       // within the past day from this ip address.
       let ipProfilingEnabled = securityHistory.enabled && securityHistory.ipProfiling &&
         securityHistory.ipProfiling.enabled
       if (ipProfilingEnabled && verified && recency === 'day') {
-        bypass = true
+        return true
```

Comment:
It's fine as is, but I just noticed, this `if` could be removed, and just end the function with `return ipProfilingEnabled && verified && recency === 'day'`